backend/database/migrations/2024_06_03_152404_create_journey_media_table.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration {
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::create("journey_media", function (Blueprint $table) {
+            $table->uuid("id")->primary();
+            $table->foreignUuid("journey_id")->constrained()->cascadeOnDelete();
+            $table->foreignUuid("user_id");//->constrained()->cascadeOnDelete();
```

Comment:
comment needed?

Reply:
Yes in the future we want to delete uploads when a user is deleted. This is already a per-requisite for that.